buildSrc/src/main/kotlin/cpg.common-conventions.gradle.kts

```diff
@@ -112,64 +110,69 @@ tasks.withType<KotlinCompile> {
     }
 }
 
-//
-// common testing configuration
-//
-tasks.test {
-    useJUnitPlatform() {
-        excludeTags("integration")
-        excludeTags("performance")
-    }
+// Configure our test suites
+@Suppress("UnstableApiUsage")
+testing {
+    suites {
+        // The default unit-test suite
+        val test by getting(JvmTestSuite::class) {
+            useJUnitJupiter()
+        }
 
-    maxHeapSize = "4048m"
-}
+        // Our integration tests
+        val integrationTest by registering(JvmTestSuite::class) {
+            description = "Runs the integration tests"
+            dependencies {
+                implementation(project())
+                implementation(testFixtures(project(":cpg-core")))
```

Comment:
The cxx frontend does not have to be listed here?

Reply:
I am not sure, if this wouldn't result in an compilation error, if the frontend is not enabled.

Reply:
No, this produces a weird error.